spec/javascript/components/applications/application_form/PaperApplicationForm.test.js

```diff
@@ -95,6 +96,43 @@ describe('PaperApplicationForm', () => {
         )
         expect(wrapper.text()).toContain('Eligibility Information')
       })
+
+      test('lenders should be filled out', async () => {
```

Comment:
maybe just update this wording to make it a bit clearer that it's testing that we can fill it out. 

maybe: "should allow lending institution and lending agent to be filled out"?